### ISSUE TITLE
DRAFT Digest Call 2 - FDD

### ISSUE THREAD

**Issue by AHolder1** [Openscapes/cohort-planning#15]
Approved/done (internal):

- [x] Anna
- [x] Julie
- [x] Rachel

---
Hi All,

Thanks for a great second session last Friday. Thank you to Ileana Fenwick for guest teaching about Data Strategies for Future Us and sharing more examples during the Q&A! Below is a light digest of Call 2, including a reminder for your Seaside Chat and additional Co-Working times before our Call 3 on Friday, October 15th. Looking forward to seeing you then!

Have a good week,

Anna, Rachel, and Julie

<img width="1293" alt="cohort-photo-2021-fdd" src="https://user-images.githubusercontent.com/5891909/136227752-63e70412-4e10-428d-bfca-473fc51c87d3.png">

## Digest: Cohort Call 2 [ 2021-FDD ]

[CohortCalls folder](https://drive.google.com/drive/folders/1Xx-G_9s6DGz4JduYQblJNuMMFaPgU7sb?usp=sharing) - contains video recordings and completed agendas
[openscapes.github.io/2021-fdd](https://openscapes.github.io/2021-fdd) - Cohort webpage

**Goals:** We discussed team culture and data strategies for future us, with special guest [Ileana Fenwick](https://twitter.com/_ileanaf), PhD Candidate, [Nye Lab](https://nye.unc.edu/), University of North Carolina

**Task for next time: Have a Seaside Chat and work on your pathway**

1. Have a Seaside Chat with your broader research group 
2. Begin your Pathway (focus on Now: your Trailhead) 
    * [Pathways spreadsheet](https://drive.google.com/open?id=1X_-qPdWDCpw2F3nZr6vZnq87guyUIGLvekm0XV2H-A8) (Make a copy in the PathwayShare Folder via File > Make a copy)
    * Note: teams can approach this differently. Maybe the team has one pathway or multiple.
    * Consider adopting (or creating/iterating) a Code of Conduct for your research group, posting publicly (on your website, on GitHub, etc.)
    * [Onboarding/how-we-work examples](https://github.com/Openscapes/how_we_work)
3. Summarize your Seaside Chat in a GitHub Issue!
    * https://github.com/Openscapes/2021-fdd/issues
    * This is a chance to keep practicing with Issues and is another way to connect across teams during our Cohort. 

4. Co-working: (optional). This is a time to come work on your own data science things, but also be able to talk things through and screenshare to get help from others. Schedule upcoming - check your emails for Calendar Invite with Zoom links.

**Slide Decks:**

These are also available publicly from [openscapes.org/series](https://openscapes.github.io/series)

* [Psychological safety](https://docs.google.com/presentation/d/1TwCyf9xicLWBfPhW9HnYQH3-mHycEyVKTm38zSg4D3Q/edit?usp=sharing)
* [Data strategies for Future Us](https://docs.google.com/presentation/d/1rv-JfJeuYhogxV6Dpn_hNDm09nfKnOMtmZpgcciI_98/edit?usp=sharing) - [Ileana Fenwick](https://twitter.com/_ileanaf), Nye Quantitative Fisheries Ecology [Lab](https://nye.unc.edu/)
* [Code of Conduct](https://docs.google.com/presentation/d/1eydm6NcrR_T2NwoMWYBWM682OW-qr755iYemVWL8cJg/edit#slide=id.p) -- we did not review this during the call, but these might be a helpful place to start for developing your own code of conduct, potentially during a seaside chat. 

**Slack**

We sent invitations to the Openscapes Slack Organization so that you can connect further with our Cohort (private channel) and with the Openscapes Community. Please introduce yourselves in the welcome channel!
* New to [Slack](https://slack.com/)? Check out this [quick start guide](https://zenodo.org/record/3763730#.YVdZ_ZrMLIV).

**A few lines from shared notes in the Agenda doc:**

There were lots of positive Seaside Chat experiences! 
* Folks were: productive, inspired, trying/learning new things together, organizing and orienting themselves to workflows and GitHub, discussing and scoping where we go from here
* Many want to dive deeper/learn/figure out how to use GitHub (Projects, Issues, Tasks, etc.) to complement or replace existing workflows in other programs (e.g. Asana, Trello, Slack, R, etc.)

Some things folks have done or seen on a team to make it psychologically safe include:
* Instead of “do you have any questions?” asking “What questions do you have?” +1 +1+1
* Being vulnerable to encourage group vulnerability +1 +1+1
* Asking anonymous questions via [pollev](https://pollev.com/home) or other means (e.g. jamboard, chat, shared docs, etc.) to encourage people speaking up or contributing to the conversation without needing to speak at all. +1 +1 +1
* Small groups and 1-on-1 conversations +1+1 +1
* Engaging everyone in the group - be sure to give everyone an opportunity to speak and contribute (instead of only hearing from the handful of folks who speak regularly) +1+1
*  Can take time before people feel comfortable to be vulnerable and share openly +1
    - YES! Building trust and vulnerability takes time; it is built/earned through little moments at a time, not given outright.

Ileana's Data strategies for Future Us presentation and discussion resonated with the group:
* Really excited about these strategies and will use them immediately in the work & looking forward to learning as we go! 
* Shout out to janitor::clean_names() & functions in {stringr} which will convert things to what you want.
    - I’ve never used clean_names(), but I find its existence thrilling.

**Comment by AHolder1** [comment]
Hi @jules32 and @rachelbaum! 

I have drafted the Digest from today's call. There are still a couple of things in brackets that we need to take care of (general summary, date/time of optional co-working meeting), but I figured we could figure that out during the seaside chat. Feel free to revise the draft digest when you get a chance, and once you're done, check the box next to your name when you're done. 

Go team! :)
